### ISSUE TITLE
Previous training script not working with calamari 2.1.1

### ISSUE THREAD

**Issue by abhikatoldtrafford** [Calamari-OCR/calamari#252]
Hi @ChWick 
I used to train calamari with the following command:
CUDA_VISIBLE_DEVICES=0 calamari-train --n_augmentations .1 --num_threads 8 --batch_size 16 --network=cnn=64:3x3,pool=2x2,cnn=128:3x3,pool=2x2,lstm=300,dropout=0.1,learning_rate=.00001 --epochs 15 --checkpoint_frequency 1 --files "./Images" --train_data_on_the_fly --validation_data_on_the_fly --early_stopping_frequency 5 --validation './validation/*.png' --output_dir "./trained_model" --weights "./trained_model/best.ckpt"

After upgrading to calamari 2.1.1, it is not working and there are a host of changes. So I am unable to 'transform' this script to the new calamari version. Can you please help in this regard? I cant even find 'arguments' in train.py so that I could change the trainign code myself. Any help wouldbe great.

**Comment by ChWick** [comment]
Have a look at the [docs](https://calamari-ocr.readthedocs.io/en/latest/doc.command-line-usage.html#calamari-train): . Your command should be:
```shell
calamari-train \
  --n_augmentations .1\
  --train.processes 8 \
  --val.processes 8 \
  --train.batch_size 16 \
  --val.batch_size 16 \
  --network=cnn=64:3x3,pool=2x2,cnn=128:3x3,pool=2x2,lstm=300,dropout=0.1\
  --learning_rate.lr .00001 \
  --trainer.epochs 15 \
  --train.images "./Images" \
  --train.preload False \
  --val.preload False \
  --early_stopping.frequency 5 \
  --val.images './validation/*.png' \
  --trainer.output_dir "./trained_model" \
  --warmstart.model "./trained_model/best.ckpt"
```

 I hope I didnt miss anything

**Comment by ChWick** [comment]
I assume that the call is working. Feel free to reopen if there are any errors or problems!
